crates/amaru-ledger/src/rules/transaction/inputs.rs

```diff
@@ -96,3 +96,318 @@ where
+                                0
+                            ),
+                            fake_output!(
+                                "704e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e749"
+                            ),
+                        ),
+                        (
+                            fake_input!(
+                                "5c6d1dba0f74e55143a751d3f92433bcfbaa63d20b909f5dde8246db9f38144c",
+                                0
+                            ),
+                            fake_output!(
+                                "704e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e749"
+                            ),
+                        ),
+                        (
+                            fake_input!(
+                                "f0b861a084a983537de82d769ddf50876ef1bbb7b504690f45bc65e65d4de8f2",
+                                0
+                            ),
+                            fake_output!(
+                                "704e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e749"
+                            ),
+                        ),
+                        (
+                            fake_input!(
+                                "878296eb7662c6a6e600f48941b3eae1e582bc05aacd0d635fcc96448a5e4e26",
+                                0
+                            ),
+                            fake_output!(
+                                "704e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e749"
+                            ),
+                        ),
+                        (
+                            fake_input!(
+                                "efd8a48f69e5270cb73b438067e1b566d39758329a45e5d6d36755f2c7326184",
+                                0
+                            ),
+                            fake_output!(
+                                "704e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e749"
+                            ),
+                        ),
+                        (
+                            fake_input!(
+                                "537de728655d2da5fc21e57953a8650b25db8fc84e7dc51d85ebf6b8ea165596",
+                                1
+                            ),
+                            fake_output!(
+                                "70f5542fa9d9c61b4dbccd71654cc5a72a9fddc9d563f503682e772ec9"
+                            ),
+                        ),
+                        (
+                            fake_input!(
+                                "47a890217e4577ec3e6d5db161a4aa524a5cce3302e389ccb22b5662146f52ab",
+                                2
+                            ),
+                            fake_output!(
+                                "605309fa786856c1262d095b89adf64fe8a5255ad19142c9c537359e41"
+                            ),
+                        ),
+                    ]),
+                },
+            ),
```

Comment:
It feels to me that these would be better provided as external JSON or CBOR files 😅, next to the serialized transactions.